[DOC-5595][DOC-5343][DOC-5250][DOC-3474] v22.2 misc deployment & operations updates (#15608)
cockroachdb/docs: _includes/v22.2/metric-names-serverless.md

`raftlog.truncated` | Number of Raft log entries truncated
`range.adds` | Number of range additions
`range.raftleadertransfers` | Number of Raft leader transfers
`range.removes` | Number of range removals
`range.snapshots.generated` | Number of generated snapshots
`range.snapshots.recv-in-progress` | Number of non-empty snapshots in progress on a receiver store
`range.snapshots.recv-queue` | Number of queued non-empty snapshots on a receiver store
`range.snapshots.recv-total-in-progress` | Number of empty and non-empty snapshots in progress on a receiver store
`range.snapshots.send-in-progress` | Number of non-empty snapshots in progress on a sender store
`range.snapshots.send-queue` | Number of queued non-empty snapshots on a sender store

> `range.snapshots.send-total-in-progress` | Number of empty and non-empty in-progress

Author: ```suggestion
`range.snapshots.send-total-in-progress` | Number of empty and non-empty in-progress on a sender store
```

> snapshots on a sender store

Author: ```suggestion
```

`range.raftleadertransfers` | Number of raft leader transfers
`range.removes` | Number of range removals
`range.snapshots.generated` | Number of generated snapshots
`range.snapshots.recv-in-progress` | Number of non-empty snapshots in progress on a receiver store
`range.snapshots.recv-queue` | Number of queued non-empty snapshots on a receiver store
`range.snapshots.recv-total-in-progress` | Number of empty and non-empty snapshots in progress on a receiver store
`range.snapshots.send-in-progress` | Number of non-empty snapshots in progress on a sender store
`range.snapshots.send-queue` | Number of queued non-empty snapshots on a sender store
`range.snapshots.send-total-in-progress` | Number of empty and non-empty in-progress

> snapshots on a sender store

Reviewer: Looks like an extra line break above this.

Author: I'm not sure what you mean. I don't see any extra breaks in the source (based on looking at the line numbers) or in https://deploy-preview-15608--cockroachdb-docs.netlify.app/docs/v22.2/ui-custom-chart-debug-page.html.

Reviewer: I'm referring to these lines: 
<img width="566" alt="image" src="https://user-images.githubusercontent.com/8740013/201785059-0dfa2445-1511-4111-855e-764d6dc8f90e.png">

`replicas.quiescent` | Number of quiesced replicas
`replicas.reserved` | Number of replicas reserved for snapshots
`replicas` | Number of replicas

> `requests.backpressure.split` | Number of backpressured writes waiting on a Range split. A Range will backpressure (roughly) non-system traffic when the range is above the configured size until the range splits. When the rate of this metric is nonzero over extended periods of time, it should be investigated why splits are not occurring.

Reviewer: ```suggestion
`requests.backpressure.split` | Number of backpressured writes waiting on a range split. A range will backpressure (roughly) non-system traffic when the range is above the configured size until the range splits. When the rate of this metric is nonzero over extended periods of time, it should be investigated why splits are not occurring.
```

`requests.slow.distsender` | Number of replica-bound RPCs currently stuck or retrying for a long time. Note that this is not a good signal for KV health. The remote side of the RPCs tracked here may experience contention, so an end user can easily cause values for this metric to be emitted by leaving a transaction open for a long time and contending with it using a second transaction.

> `requests.slow.lease` | Number of requests that have been stuck for a long time acquiring a lease. This gauge registering a nonzero value usually indicates range or replica unavailability, and should be investigated. In the common case, we also expect to see 'requests.slow.raft' to register a nonzero value, indicating that the lease requests are not getting a timely response from the replication layer.

Reviewer: Elsewhere the metrics names don't have enclosed punctuation, so suggesting this for consistency

```suggestion
`requests.slow.lease` | Number of requests that have been stuck for a long time acquiring a lease. This gauge registering a nonzero value usually indicates range or replica unavailability, and should be investigated. In the common case, we also expect to see requests.slow.raft to register a nonzero value, indicating that the lease requests are not getting a timely response from the replication layer.
```
